internal/exec/stages/files/passwd.go

```diff
@@ -33,13 +52,18 @@ func (s *stage) createPasswd(config types.Config) error {
 	// to be safe, just blanket mark all passwd-related files rather than
 	// trying to make it more granular based on which executables we ran
 	if len(config.Passwd.Groups) != 0 || len(config.Passwd.Users) != 0 {
-		s.relabel(
-			"/etc/passwd*",
+		deglobbed, err := s.deglobList("/etc/passwd*",
 			"/etc/group*",
 			"/etc/shadow*",
 			"/etc/gshadow*",
 			"/etc/subuid*",
-			"/etc/subgid*",
+			"/etc/subgid*")
+		if err != nil {
+			return err
+		}
+
```

Comment:
A short comment here too on why we're doing this before adding to the list would be helpful.